ConsumerApi.Sdk/Endpoints/Files/FilesEndpoint.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Backbone.ConsumerApi.Sdk.Endpoints.Common;
+using Backbone.ConsumerApi.Sdk.Endpoints.Common.Types;
+using Backbone.ConsumerApi.Sdk.Endpoints.Files.Types;
+
+namespace Backbone.ConsumerApi.Sdk.Endpoints.Files;
+
+public class FilesEndpoint(EndpointClient client) : Endpoint(client)
+{
+    public async Task<ConsumerApiResponse<CreateFileResponse>> UploadFile(CreateFileRequest request) => await _client
+        .Request<CreateFileResponse>(HttpMethod.Post, "Files")
+        .Authenticate()
+        .WithMultipartForm(request.ToMultipartContent())
+        .Execute();
+
+    public async Task<ConsumerApiResponse<List<FileMetadata>>> ListFileMetadata(PaginationFilter? pagination = null)
```

Comment:
You should introduce Response classes for all `List*` methods. I made an example for the `ListDevices` method.

Reply:
👍